docs/introduction/framework-extensibility.md

```diff
@@ -13,22 +13,64 @@ as well as a list of customizations that are not (and will not be) possible at a
 * [Customizing database migrations](/docs/introduction/database-migrations.md)
     * adding a new migration as well as skipping and reordering the existing ones
 * Configuring the smoke tests (see [`RouteConfigCustomization`](/project-base/tests/ShopBundle/Smoke/Http/RouteConfigCustomization.php) class)
-    * *Note: This is now achievable as the configuration class is located in the open box project-base. 
+    * *Note: This is now achievable as the configuration class is located in the open box project-base.
     However, that makes the upgrading of the component harder so the configuration is planned to be re-worked.*
 * [Implementing custom product feed or modifying an existing one](/docs/introduction/product-feeds.md)
 * [Extending the application using standard Symfony techniques](https://symfony.com/doc/current/bundles/override.html)
     * e.g. overriding Twig templates, routes, services, ...
 * [Adding a new advert position](/docs/cookbook/adding-a-new-advert-position.md) to be used in the administration section *Marketing > Advertising system*
 * open-box modifications in `project-base`
     * e.g. adding new entities, changing the FE design, customization of FE javascripts, adding new FE pages (routes and controllers), ...
+* [Hiding the existing features and functionality](https://github.com/shopsys/demoshop/pull/13)
+
+## What is achievable with additional effort
+* [Extending factories and controllers](https://github.com/shopsys/demoshop/commit/898d111879aef40196f79ac763373560f44aef59#diff-1b3bd68670cd376165cdc6cfc634f24f)
+* Adding custom javascript into administration
+* [Adding form option into existing form](https://github.com/shopsys/demoshop/commit/898d111879aef40196f79ac763373560f44aef59#diff-3293b000b06ad6c0280341584c4d661d)
+* [Extending administration theme](https://github.com/shopsys/demoshop/commit/d0e0eaaa2eeac5e1c90d8a29be5c827c4a067b9f)
+* [Changing an entity association](https://github.com/shopsys/demoshop/commit/9931083ea37ad611568e32bc1a9c8cf203401809)
+    * this change is complicated and potentially dangerous
 
 ## Which issues are going to be addressed soon
 * Extending data fixtures (including performance data fixtures)
 * Extending data grids in the administration
 * Extending all forms in the administration without the need of the template overriding
-* Hiding the existing features and functionality
+* Extending classes like Repositories don't lead to change of project-base tests
+* Incorrectly extended entities cause clear exceptions
 
 ## What is not achievable
 * Removing an attribute from a framework entity
-* Changing a data type of an entity attribute or association
-* Removing existing entities and features
\ No newline at end of file
+* Changing a data type of an entity attribute
```

Comment:
:thinking: when changing a data type of association is possible, should not it be possible for entity attributes as well? :slightly_smiling_face: